### ISSUE TITLE
Athena quantile support lost

### ISSUE THREAD

**Issue by kuhnen** [fivetran/great_expectations#4851]
**Describe the bug**
Unfortunately, we lost the feature of having quantiles expectations when using Athena or Presto.
The main reason is that presto has a different method to calculate these numbers:

https://prestodb.io/docs/current/functions/aggregate.html#approx_percentile

**To Reproduce**
Connect to any Athena table and try to  use the "expect_column_quantile_values_to_be_between"

**Expected behavior**
you should see this error:

```
great_expectations.exceptions.exceptions.MetricResolutionError: (pyathena.error.DatabaseError) An error occurred (InvalidRequestException) when calling the StartQueryExecution operation: line 1:43: mismatched input '('. Expecting: 'BY'
[SQL: SELECT percentile_disc(%(percentile_disc_1)s) WITHIN GROUP (ORDER BY "amount" ASC) AS "anon_1", percentile_disc(%(percentile_disc_2)s) WITHIN GROUP (ORDER BY "amount" ASC) AS "anon_2", percentile_disc(%(percentile_disc_3)s) WITHIN GROUP (ORDER BY "amount" ASC) AS "anon_3", percentile_disc(%(percentile_disc_4)s) WITHIN GROUP (ORDER BY "amount" ASC) AS "anon_4", percentile_disc(%(percentile_disc_5)s) WITHIN GROUP (ORDER BY "amount" ASC) AS "anon_5"
FROM "ge_temp_e291c2e0"]
[parameters: {'percentile_disc_1': 0.05, 'percentile_disc_2': 0.25, 'percentile_disc_3': 0.5, 'percentile_disc_4': 0.75, 'percentile_disc_5': 0.95}]
(Background on this error at: https://sqlalche.me/e/14/4xp6)
```
 - Great Expectations Version: [e.g. 0.15.0]

**Additional context**

We already implemented this on the past to Great expectations.
Either me @kuhnen   or @RicardoPedrotti  :). 

Thanks a lot for you support

**Comment by RicardoPedrotti** [comment]
These were the pull requests where the _expect_column_quantile_values_to_be_between_ calculations on Athena/Presto were introduced to Great Expectations:
https://github.com/great-expectations/great_expectations/pull/2521
https://github.com/great-expectations/great_expectations/pull/2544

**Comment by talagluck** [comment]
Thanks all! We will review and be in touch.

**Comment by austiezr** [comment]
Closed by #5114
